node-wasm/src/client.rs

```diff
@@ -32,6 +34,8 @@ pub struct WasmNodeConfig {
     /// A list of bootstrap peers to connect to.
     #[wasm_bindgen(getter_with_clone)]
     pub bootnodes: Vec<String>,
+    /// Custom syncing window size. Pruning starts one hour after syncing window end.
```

Comment:
could you explain there shortly what syncing window is?

Reply:
Used the description from cli everywhere, as it's most descriptive :+1: